features/overlay.yml

```
@@ -0,0 +1,5 @@
+name: overlay
+description: The `overlay` CSS property, used as an `allow-discrete` CSS transition, prevents a top layer element, such as a popover or a `<dialog>`, from being removed from the top layer before it has finished animating. You can't set the value of the `overlay` property, only use it as transition property.
```

I think this description is really good! I have the teeniest, tiniest style nit.

```suggestion
description: The `overlay` CSS property, used as an `allow-discrete` CSS transition, prevents a top layer element, such as a popover or a `<dialog>`, from being removed from the top layer before it has finished animating. You can't set the value of the `overlay` property; only use it as transition property.
```